### PR TITLE
Upload photo

### PR DESCRIPTION
**Pull request by rostyk1331**

### CODE REVIEW

File: server/src/controllers/profile.controllers/handlePhoto.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    if (file.mimetype === 'image/jpeg' || file.mimetype === 'image/png') {
+        cb(null, true);
+    } else {
+        cb(new Error('Please, choose a valid image'), false);
+    }
+};
+
+export const upload = multer({
+    storage,
+    limits: { fileSize: 1024 * 1024 * 5 },
```
Comment (InnaKutsik) [inline server/src/controllers/profile.controllers/handlePhoto.ts]: is not 5Mb a lot?

File: server/src/controllers/profile.controllers/handlePhoto.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    fileFilter,
+});
+
+const handlePhotoChange = async (req: Request): Promise<any> => {
+    const user: any = await User.findOneAndUpdate(
```
[the remark this answers is not in the document]
Comment (rostyk1331) [reply server/src/controllers/profile.controllers/handlePhoto.ts]: Done.

File: server/src/controllers/profile.controllers/handlePhoto.ts (hunk 1 above)
Comment (rostyk1331) [reply server/src/controllers/profile.controllers/handlePhoto.ts]: Changed to 2Mb.

File: client/src/components/Alert/Alert.tsx [hunk 3]
```diff
@@ -23,7 +23,7 @@ const Alert: React.FunctionComponent<any> = ({
   color,
   cleanAlert,
 }: IAlertProps): any => message &&
-    <UncontrolledAlert color={color} onclick={(): void => {cleanAlert(); }}>{message}</UncontrolledAlert>
+    <UncontrolledAlert color={color} onclick={(): void => {cleanAlert(); }}>{message}</UncontrolledAlert>;
```
Comment (InnaKutsik) [inline client/src/components/Alert/Alert.tsx]:
```
onClick={cleanAlert}
```

File: server/.env.example [hunk 4]
```diff
@@ -15,3 +15,8 @@ GOOGLE_CLIENT_SECRET = 'Xtktwt99ZXrL8rIVt23aStU2'
 GOOGLE_CALLBACK_URL = 'http://localhost:8080/auth/google/callback'
 
 EXPIRES='3600'
+
+AWS_SECRET_ACCESS_KEY = 
+AWS_ACCESS_KEY_ID = 
+AWS_REGION =
```
Comment (InnaKutsik) [inline server/.env.example]: ?

File: client/src/components/Alert/Alert.tsx (hunk 3 above)
Comment (rostyk1331) [reply client/src/components/Alert/Alert.tsx]: Done.

File: server/.env.example (hunk 4 above)
Comment (rostyk1331) [reply server/.env.example]: Not sure how to change it. Changed to AWS_SECRET_ACCESS_KEY =AWS_SECRET_ACCESS_KEY

**Pull request merged**